M2/Macaulay2/m2/matrix2.m2

```diff
@@ -351,8 +352,11 @@ quotient(Matrix, Matrix) := Matrix => opts -> (f, g) -> (
 addHook((quotient, Matrix, Matrix), Strategy => Default,
     -- Note: this strategy only works if the remainder is zero, i.e.:
     -- homomorphism' f % image Hom(source f, g) == 0
-    -- TODO: should we pass MinimalGenerators => false to Hom and homomorphism'?
-    (opts, f, g) -> map(source g, source f, homomorphism(homomorphism' f // Hom(source f, g))))
+    (opts, f, g) -> (
+	opts = new OptionTable from {
+	    DegreeLimit       => opts.DegreeLimit,
+	    MinimalGenerators => opts.MinimalGenerators };
+	map(source g, source f, homomorphism(homomorphism'(f, opts) // Hom(source f, g, opts)))))
```

Comment:
Should we add any tests for these changes?

Reply:
Some tests are included in a different package which may or may not make it to this release. I think existing tests are sufficient for now.